src/calculateRentalCost.js

```diff
@@ -4,7 +4,30 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
+  const costCar = 40;
+  const threeDaysDiscount = 20;
+  const sevenDaysDiscount = 50;
+
+  if (days < 3) {
+    sum = costCar * days;
+
+    return sum;
+  }
+
+  if (days >= 3 && days <= 6) {
+    sum = costCar * days - threeDaysDiscount;
+
+    return sum;
+  }
+
+  if (days >= 7) {
+    sum = costCar * days - sevenDaysDiscount;
+
+    return sum;
+  }
+
+  return sum;
 }
```

Comment:
Consider using `const` instead of `let` for the `sum` variable since it's not being reassigned, which adheres to the checklist item 3 preferring `const` over `let` where possible.